acceptance/bundle/unitycatalog/capture_schema_dependency/dlt_schema/out.json

```diff
@@ -0,0 +1,28 @@
+{
+  "pipeline1": {
+    "catalog": "catalog1",
+    "schema": "${resources.schemas.schema1.name}"
```

Comment:
Is it expected that this is not resolved to the name? We have the name, right?

Reply:
> Is it expected that this is not resolved to the name? We have the name, right?

The terraform layer in the stack resolves any values with the `resources` prefix. `bundle/deploy/terraform/interpolate.go` is the translation layer between these references and syntax that terraform understands.

Reply:
Does it work for 'name' as well? I thought it's for looking up ids.

In any case, could you add this context as a comment on script?

Reply:
Should we also add 'bundle summary' to see the resolution in action?